src/extensions/screenshooter.rs

```diff
@@ -0,0 +1,27 @@
+use wayland_sys::server::wl_display as wl_server_display;
+use wlroots_sys::{wl_display, wlr_screenshooter_manager, wlr_screenshooter_manager_create, 
+    wlr_screenshooter_manager_destroy};
```

Comment:
Make this line up with the previous line (e.g. the `w` in `wlr` should line up with `wl`. See the other files for examples, my editor does this automatically).